### ISSUE TITLE
LocalStorage: vuex plugins error

### ISSUE THREAD

**Issue by pinghe** [quasarframework/quasar#1680]
### Software version

Quasar: v0.15.5
OS: Windows
Node: 9.6.1
NPM: 5.6.0
Browsers: chrome

quasar.conf.js
````
    framework: {
      ...
      plugins: [
        'LocalStorage',
        'Notify'
      ]
    },
````

store/index.js
````
import Vue from 'vue'
import Vuex from 'vuex'

import { LocalStorage } from 'quasar'

Vue.use(Vuex)

const testPlugin = store => {
   if (LocalStorage.has('testkey')) {
   } else {
   }

  store.subscribe((mutation, state) => {
  })
}

const store = new Vuex.Store({
  strict: true,

  state: {
  },
  mutations: {

  },
  modules: {
  },
  plugins: [
    testPlugin
  ]
})

export default store
````

error messages:
````
index.js?e3b1:14 Uncaught TypeError: __WEBPACK_IMPORTED_MODULE_3_quasar__.b.has is not a function
````

**Comment by rstoenescu** [comment]
Fixing this corner-case. Vuex store code runs before Quasar plugins get installed.

There's one easy fix from CLI but it needs to turn both router and store exports as functions than direct reference to Router and Store. But won't do this as it adds a bit of overhead and it's a breaking change.

The fix I'm going to write is to install plugins on demand automatically.

**Comment by rstoenescu** [comment]
Fix available in 0.15.6.
